README.md

```diff
@@ -5,6 +5,9 @@ Add you name and github profile here. Have fun with MERGE CONFLICTS (if you were
 - [Aaditya Agarwal](https://github.com/psy-duck)
 - [Palak Gupta](https://github.com/Palakfs)
 - [Aditya Karwa](https://github.com/AdityaK-github)
+- [Yash Gupta](https://github.com/Yash-g2310)
+- [Ayush Dhar Dubey](https://github.com/AyushDharDubey)
+- [Rayyan Khan](https://github.com/rkcoder101)
```

Comment:
@Yash-g2310 please remove these 2 lines, as these are duplicate

Reply:
i have removed those two lines and resolved all conflicts